core/src/consensus/tendermint/message.rs

```diff
@@ -127,20 +127,28 @@ impl Decodable for TendermintMessage {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, Clone, Hash, Default, RlpDecodable, RlpEncodable)]
+pub struct VoteOn {
+    pub step: VoteStep,
```

Comment:
It's a shame to have `step.step` because VoteStep has a field named `step`.
But I couldn't think a good alternative.

Reply:
I agree.